tests/rules/javascript/test.no_implied_eval.js

```diff
@@ -136,30 +135,30 @@ describe('no_implied_eval', () => {
       message: [expectedErrorMessage],
       description: [NO_IMPLIED_EVAL.description],
     },
+
+    // ??? there is also 'gives the correct node when dealing with nesting' in the source
```

Comment:
There is also code for 'gives the correct node when dealing with nesting' in the source